analysis/webservice/algorithms/doms/ResultsStorage.py

```diff
@@ -192,7 +192,9 @@ def __insertResults(self, execution_id, results):
         inserts = []
 
         for result in results:
-            inserts.extend(self.__prepare_result(execution_id, None, result, insertStatement))
+            # 'PRIMARY' arg since primary values cannot have primary_value_id be null anymore
+            # Secondary matches are prepped recursively from this call
+            inserts.extend(self.__prepare_result(execution_id, 'PRIMARY', result, insertStatement))
```

Comment:
Can we change `'PRIMARY'` to None and check against that instead?

Reply:
It actually used to be `None`, but since it's now part of the PK, it can't be `None` so I used `'PRIMARY'` as a stand-in.

Reply:
Ohhh gotcha. This is literally being inserted into the DB as a stand-in. Thanks!